src/ResourceManagement/Search/AdminKeysImpl.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+namespace Microsoft.Azure.Management.Search.Fluent
```

Comment:
@martinsawicki
This class is the required implementation for the AdminKeys interface which is a wrapper for the *Inner respective class; in Fluent we do not want to expose the *Inner directly to the user.